modules/users/client/services/password-validator.client.service.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// PasswordValidator service used for testing the password strength
+angular.module('users').factory('PasswordValidator', ['$window',
+  function ($window) {
+    var owaspPasswordStrengthTest = $window.owaspPasswordStrengthTest;
+
+    return {
+      getResult: function (password) {
+        var result = owaspPasswordStrengthTest.test(password);
+        return result;
+      },
+      getPopoverMsg: function () {
+        var popoverMsg = "Please enter a passphrase or password with greater than 10 characters, numbers, lowercase, upppercase, and special characters.";
```

Comment:
Thanks @simison , I didn't see this show after running `grunt jshint` on that branch. Output from the command:
```
grunt jshint
Running "jshint:all" (jshint) task
>> 105 files lint free.

Done, without errors.
```
Any thoughts on why I don't get the errors?
I can submit another PR to correct this though...